tooling/strace_analysis.sh

```diff
@@ -173,9 +173,16 @@ processFiles() {
         filePath="$(readlink -f "$file")"
         non_pkg=false
 
+        package_query="rpm -qf"
+        # Alpine uses apk info --who-owns
+        cat /etc/os-release | grep "Alpine Linux" 2>&1>/dev/null
+        isAlpine=$(echo $?)
+        if [[ "$isAlpine" == "0" ]]; then
```

Comment:
Not sure this will work as $(echo $?) might be run in a different shell,
can we use this instead?
```
if grep "Alpine Linux" /etc/os-release 2>&1>/dev/null; then
```

Reply:
`2>&1>/dev/null` won't work. Use `>/dev/null 2>&1` or `>/dev/null 2>/dev/null`